FinniversKit/Sources/Components/FrontPageTransactionView/FrontPageTransactionView.swift

```diff
@@ -0,0 +1,201 @@
+        imageView.layer.cornerRadius = cornerRadius
+        imageView.clipsToBounds = true
+        return imageView
+    }()
+    
+    private lazy var verticalStack: UIStackView = {
+        let stack = UIStackView(withAutoLayout: true)
+        stack.axis = .vertical
+        stack.alignment = .leading
+        stack.distribution = .fillProportionally
+        return stack
+    }()
+    
+    private lazy var backgroundContainer: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.backgroundColor = .white
+        view.layer.cornerRadius = cornerRadius
+        return view
+    }()
+    
+    private lazy var imageContainerView: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.backgroundColor = .bgColor
+        view.layer.cornerRadius = cornerRadius
+        view.layer.borderWidth = 1
+        view.layer.borderColor = UIColor.black.withAlphaComponent(0.1).cgColor
+        return view
+    }()
+    
+    private lazy var smallShadowView: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.dropShadow(
+            color: .shadowColor,
+            opacity: 0.24,
+            offset: CGSize(width: 0, height: 1),
+            radius: 1
+        )
+        return view
+    }()
+
+    private lazy var largeShadowView: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.dropShadow(
+            color: .shadowColor,
+            opacity: 0.16,
+            offset: CGSize(width: 0, height: 1),
+            radius: 5
+        )
+        return view
+    }()
+    
+    private lazy var horizontalStack: UIStackView = {
+        let stack = UIStackView(withAutoLayout: true)
+        stack.axis = .horizontal
+        stack.distribution = .fillProportionally
+        stack.spacing = .spacingXL
+        return stack
+    }()
+
+    private lazy var headerLabel: UILabel = {
+        let label = Label(withAutoLayout: true)
+        label.font = .bodyStrong
+        label.textColor = .textPrimary
+        return label
+    }()
+
+    private lazy var containerStack: UIStackView = {
+        let stack = UIStackView(withAutoLayout: true)
+        stack.axis = .vertical
+        stack.distribution = .fillProportionally
+        stack.spacing = .spacingS
+        return stack
+    }()
+
+
+    private(set) var viewModel: FrontPageTransactionViewModel?
+    private var imageDatasource: RemoteImageViewDataSource?
+    public weak var delegate: FrontPageTransactionViewDelegate?
+    
+    //MARK: - Lifecycle
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        setup()
+    }
+}
+
+//MARK: - Private functions
+extension FrontPageTransactionView {
+    private func setup() {
+        imageContainerView.addSubview(imageView)
+        imageView.fillInSuperview()
+        // addSubview(largeShadowView)
```

Comment:
This comment should probably be removed😄

Reply:
done  😄